.github/workflows/ci_app.yml

```diff
@@ -30,7 +30,8 @@ jobs:
         image: redis:3.2-alpine
         ports: ["6379:6379"]
     env:
-      DB_DATABASE: app-test
+      DB_TEST_DATABASE: app-test
+      DB_DATABASE: app
```

Comment:
which one is correct?

Reply:
I'm not sure about it, but I think current version makes more sense becasuse we have both DB_TEST_DATABASE and DB_DATABASE in application.yml

Reply:
can you try removing `DB_DATABASE` and check if CI is still works?

Reply:
CI stops working after removing either DB_DATABASE or DB_TEST_DATABASE